src/content/developers/tutorials/academic-certificates-with-truffle/index.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Academic certificates with Truffle
+description: Ethereum smart contract tutorial. Learn how to develop and deploy a contract that issues academic certificates. Built using Truffle suite.
+author: Ake Gaviar
+lang: en
+tags: ["chainstack", "truffle", "solidity"]
+skill: intermediate
+published: 2023-01-31
+source: Chainstack
+sourceUrl: https://docs.chainstack.com/docs/ethereum-tutorial-academic-certificates-with-truffle
+---
+
+In this tutorial, you will:
+
+- Create a DApp that generates an academic certificate.
+- Deploy the DApp on a public Ethereum node using Chainstack.
+
+The contract and the Truffle configuration are in the [GitHub repository](https://github.com/chainstack/ethereum-certificates-tutorial).
+
+## Prerequisites
+
+- [Chainstack account](https://console.chainstack.com/user/login) to deploy an Ethereum node
+- [Truffle Suite](https://trufflesuite.com/docs/truffle/how-to/install/) to create and deploy contracts
+
+## Overview
+
+To get from zero to a deployed DApp on the Ethereum Goerli testnet, do the following:
+
+1. With Chainstack, create a <<glossary:public chain project>>.
+2. With Chainstack, join the Ethereum Goerli testnet.
+3. With Chainstack, access your Ethereum node credentials.
+4. With Truffle, create and compile the DApp contract.
+5. With Truffle, deploy the contract to your local development network.
+6. With Truffle, interact with the contract on your local development network.
+7. With Truffle, create and run the contract test.
+8. With Truffle, deploy the contract to your Ethereum node running with Chainstack.
+
+## Step-by-step
+
+### Create a public chain project
+
+See [Create a project](doc:manage-your-project#create-a-project).
+
+### Join the Ethereum Goerli testnet
+
+See [Join a public network](doc:manage-your-networks#join-a-public-network).

```

Comment:
Part of the reason for this is there are a lot of broken links that you will need to go through and fix. For example here. 

Right now these tutorials are broken if we keep them internal.

I would prefer this be an external one as well given its tightly coupled to a project.